src/odemis/acq/fastem.py

```diff
@@ -546,30 +547,67 @@ def acquireTiledArea(stream, stage, area, live_stream=None):
 
     est_dur = estimateTiledAcquisitionTime(sem_stream, stage, area)
     f = model.ProgressiveFuture(start=time.time(), end=time.time() + est_dur)
+
+    # Connect the future to the task and run it in a thread.
+    # _run_overview_acquisition is executed by the executor and runs as soon as no other task is executed
     _executor.submitf(f, _run_overview_acquisition, f, sem_stream, stage, area, live_stream)
 
     return f
 
 
 def estimateTiledAcquisitionTime(stream, stage, area):
+    """
+    Estimate the time needed to acquire a full overview image. Calculate the
+    number of tiles needed for the requested area based on set dwell time and
+    resolution (number of pixels).
+
+    :param stream: (SEMstream) The stream used for the acquisition.
+    :param stage: (actuator.MultiplexActuator) The stage in the sample carrier coordinate system.
+        The x and y axes are aligned with the x and y axes of the ebeam scanner.
+    :param area: (float, float, float, float) minx, miny, maxx, maxy coordinates of the overview region.
+
+    :return: The estimated total acquisition time for the overview image in seconds.
+    """
+
     # TODO: fix function to limit the acquisition area so that the FoV is taken into account.
```

Comment:
@pieleric

Reply:
I will leave it in.

Reply:
@Sabrina85 , I think you should drop the TODO. Just replace by a comment explaining why we cannot rely on the standard estimateTiledAcquisitionTime() function. Something like "estimateTiledAcquisitionTime() cannot be used for a couple of reasons: firstly because the e-beam settings are not the current ones, but the one from the fastem_conf. Also, the xt_client acquisition time is quite a bit longer than the settings suggests. So we have ad-hoc method here."